### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by hoshion** [MorphiaOrg/morphia#1812] [Ideas]
Many developers use Morphia while working with MongoDB and Java. It is a good reason for creating Discord server: people can exchange their ideas, issues and enhancements in Morphia with each other. These community and it's support may be a good motivation for keeping on updating this fantastic project.

**Comment by felixxxxxs** [comment]
Great idea. Especially since the documentation is not that comprehensive (imho), a discord server could be a great way to exchange with other developers using Morphia.

**Comment by evanchooly** [comment]
I'm not 100% sure this is a good idea but it's worth a shot.  :) 

 https://discord.gg/Sb6THwER

**Comment by evanchooly** [comment]
I think that invite expires but the permanent link should be [here](https://discordapp.com/channels/940011609047335002/940011609047335007).  

I'm a discord a newb so that link might need some work.

**Reply by yugalsaluja** [reply]
Hi @evanchooly Apparently this link doesn't work. Is this community active on discord, I need help with migrating from morohia 1.3.x to 2.2.x. Please provide me the invite link if the channel exists.

**Reply by evanchooly** [reply]
that link expired but here's a new one that won't expire:  https://discord.gg/saZsJescBa

Activity has been low lately but i'm always there (and increasingly on other discord servers) so hopefully it'll pick a bit.
